DfE.Hexagonal.Model/School.cs

```diff
@@ -0,0 +1,38 @@
+﻿using DfE.Hexagonal.Common.Validation;
+
+namespace DfE.Hexagonal.Model
+{
+    public class School : IValidatable
+    {
+        public Guid Id { get; set; }
+        public string Name { get; set; }
+        public string Description { get; set; }
+        public DateTime OpenDate { get; set; }
+
+        public ValidationResult Validate()
+        {
+            bool isValid = true;
+            var errors = new Dictionary<string, string>();
+
+            if (Id.Equals(Guid.Empty))
+            {
+                isValid = false;
+                errors.Add("Id", "Id must have a value");
+            }
+
+            if (String.IsNullOrEmpty(Name))
```

Comment:
Shall we use `string.IsNullOrEmpty` (lowercase `s`) for these checks?

Reply:
Force of habit on my part but yes, we've also said to use the primitive not in-built objects